tests/s3/versioning/test_delete_object_tagging.py

```diff
@@ -0,0 +1,163 @@
+                       object_name=self.object_name, versions_dict=versions, is_unversioned=True)
+        last_v = versions[self.object_name]["version_history"][-1]
+        LOGGER.info("Step 1: Successfully uploaded object %s before enabling versioning on  "
+                    "bucket %s with version ID %s", self.object_name, self.bucket_name, last_v)
+        LOGGER.info("Step 2: Perform PUT Bucket versioning with status as Enabled on %s",
+                    self.bucket_name)
+        resp = self.s3_ver_obj.put_bucket_versioning(bucket_name=self.bucket_name)
+        assert_utils.assert_true(resp[0], resp)
+        LOGGER.info("Step 2: Performed PUT Bucket versioning with status as Enabled on %s",
+                    self.bucket_name)
+        LOGGER.info("Step 3: Perform PUT Object Tagging for %s with a tag key-value pair",
+                    self.object_name)
+        put_object_tagging(s3_tag_test_obj=self.s3_tag_obj, s3_ver_test_obj=self.s3_ver_obj,
+                           bucket_name=self.bucket_name, object_name=self.object_name,
+                           version_tag=ver_tag)
+        LOGGER.info("Step 3: Performed PUT Object Tagging for %s with a tag key-value pair",
+                    self.object_name)
+        LOGGER.info("Step 4: Perform GET Object Tagging for %s with versionId=%s",
+                    self.object_name, last_v)
+        put_tag = ver_tag[self.object_name][last_v][-1]
+        resp = self.s3_ver_obj.get_obj_tag_ver(bucket_name=self.bucket_name,
+                                               object_name=self.object_name, version=last_v)
+        assert_utils.assert_true(resp[0], resp)
+        get_tag = resp[1]['TagSet'][0]
+        assert_utils.assert_equal(get_tag, put_tag, "Mismatch in tag Key-Value pair."
+                                                    f"Expected: {put_tag} \n Actual: {get_tag}")
+        LOGGER.info("Step 4: Performed GET Object Tagging for %s with versionId=%s is %s",
+                    self.object_name, last_v, get_tag)
+        LOGGER.info("Step 5: Perform DELETE Object Tagging for %s with versionId=%s",
+                    self.object_name, last_v)
+        resp = self.s3_ver_obj.delete_obj_tag_ver(bucket_name=self.bucket_name,
+                                                  object_name=self.object_name, version=last_v)
+        assert_utils.assert_true(resp[0], resp)
+        LOGGER.info("Step 5: Performed DELETE Object Tagging for %s with versionId=%s",
+                    self.object_name, last_v)
+        LOGGER.info("Step 6: Perform GET Object Tagging for %s with versionId=%s",
+                    self.object_name, last_v)
+        resp = self.s3_ver_obj.get_obj_tag_ver(bucket_name=self.bucket_name,
+                                               object_name=self.object_name, version=last_v)
+        assert_utils.assert_true(resp[0], resp)
+        # For null version ID, expecting "TagSet": []
```

Comment:
null or no version id?

Reply:
Version ID will be NULL only. but its tag will be [] after delete